.codebuild/canary_workflow.yml

```diff
@@ -0,0 +1,59 @@
+ version: 0.2
+env:
+  shell: bash
+  compute-type: BUILD_GENERAL1_LARGE
+batch:
+  fast-fail: false
+  build-graph:
+    - identifier: build_linux
+      buildspec: .codebuild/build_linux.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+    - identifier: test
+      buildspec: .codebuild/test.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+      depend-on:
+        - build_linux
+    - identifier: publish_to_local_registry
+      buildspec: .codebuild/publish_to_local_registry.yml
+      env:
+        compute-type: BUILD_GENERAL1_MEDIUM
+      depend-on:
+        - build_linux
+    - identifier: build_app_swift_us_east_1
+      buildspec: .codebuild/run_ios_modelgen_e2e_test.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TEST_SUITE: src/__tests__/build-app-swift.test.ts
+          CLI_REGION: us-east-1
+      debug-session: true
+      depend-on:
+        - publish_to_local_registry
+    - identifier: build_app_android
+      buildspec: .codebuild/run_android_modelgen_e2e_test.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TEST_SUITE: src/__tests__/build-app-android.test.ts
+          CLI_REGION: us-east-2
+      debug-session: true
+      depend-on:
+        - publish_to_local_registry
+    - identifier: build_app_ts
+      buildspec: .codebuild/run_e2e_tests.yml
+      env:
+        compute-type: BUILD_GENERAL1_LARGE
+        variables:
+          TEST_SUITE: src/__tests__/build-app-ts.test.ts
+          CLI_REGION: ap-southeast-1
+      debug-session: true
+      depend-on:
+        - publish_to_local_registry
+    - identifier: cleanup_e2e_resources
+      buildspec: .codebuild/cleanup_e2e_resources.yml
+      env:
+        compute-type: BUILD_GENERAL1_MEDIUM
+      depend-on:
+        - publish_to_local_registry
```

Comment:
This will start running `cleanup_e2e_resources` while the other jobs are running (once `publish_to_local_registry` completes), right? Is that intentional?

Reply:
My thought was if any test fails atleast the `cleanup_e2e_resources` run but I guess I ran the build again and it executes it before which is expected and it will leave the tests after it so I will update its dependency to the android test.